### PR TITLE
Add stats for hapi

### PR DESCRIPTION
**Pull request by rands0n**
I am adding one by one :)

:+1: ?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -132,10 +132,33 @@ <h2 class="page-headline">MVC frameworks</h2>
                 [<a href="http://github.com/azat-co/rest-api-express" title="REST API Example">rest api</a>]
                 </div>
               </li>
-              <li><i class="icon-check"></i> <a href="http://spumko.github.io/">hapi</a>
-               [<a href="https://github.com/azat-co/practicalnode/tree/master/ch8" title="REST API Example">rest api</a>]
+              <li>
+              	<i class="icon-check"></i> <a href="http://spumko.github.io/">hapi</a>
+								<iframe src="http://ghbtns.com/github-btn.html?user=hapijs&repo=hapi&type=watch&count=true" allowtransparency="true" frameborder="0" scrolling="0" width="110" height="20"></iframe>
```
Comment (azat-co) [inline index.html:137]: what's up with this indentation... can it be on 2 space? :)

**Comment by azat-co** [comment]
@randsonjs thank you! while you are there, would you mind adding `[GitHub]  [NPM]` and examples (just google the best or if they have official examples link to that) to the frameworks that don't have them?

File: index.html (hunk 1 above)
Comment (rands0n) [inline index.html:137]: Yes @azat-co. Is 2 space. But I dont know what happening :(

**Comment by rands0n** [comment]
For now, I will add `Github` and `NPM`. Once finished I will add all the other related things as you mentioned :)

**Pull request merged**